training/benchmarks/glm/pytorch/train/trainer.py

```diff
@@ -136,8 +136,9 @@ def train_one_epoch(self, dataloader):
                     self.config.gradient_accumulation_steps) / step_total_time
                 other_state["seq/s"] = sequences_per_second
 
-            loss_scale = self.optimizer.loss_scaler.loss_scale
-            other_state['loss_scale'] = loss_scale
+            if hasattr(self.optimizer, 'loss_scaler'):
+                loss_scale = self.optimizer.loss_scaler.loss_scale
+                other_state['loss_scale'] = loss_scale
 
```

Comment:
the optimizer doesn't have a loss scaler when running a fp32 training

Reply:
LGTM